src/libsyntax/ext/tt/macro_parser.rs

```diff
@@ -509,6 +509,7 @@ fn may_begin_with(name: &str, token: &Token) -> bool {
     }
 
     match name {
+        "str" => token.can_begin_str(),
```

Comment:
All the previous matchers went through RFC process.
Well, at least I'm pretty sure this addition should not be hidden in a PR named "Ignore panic functions in backtraces".